src/main/java/com/synopsys/integration/detect/configuration/DetectProperties.java

```diff
@@ -1703,8 +1703,8 @@ private DetectProperties() {
         EnumProperty.newBuilder("detect.blackduck.scan.mode", BlackduckScanMode.INTELLIGENT, BlackduckScanMode.class)
             .setInfo("Detect Scan Mode", DetectPropertyFromVersion.VERSION_6_9_0)
             .setHelp(
-                "Set the Black Duck scanning mode of Detect",
-                "Set the scanning mode of Detect to control how Detect will send data to Black Duck. RAPID will not persist the results and disables select Detect functionality for faster results. INTELLIGENT persists the results and permits all features of Detect."
+                "Set the Black Duck scanning mode of Detect.  The keyword EPHEMERAL is deprecated, use STATELESS instead",
+                "Set the scanning mode of Detect to control how Detect will send data to Black Duck. RAPID will not persist the results and disables select Detect functionality for faster results. INTELLIGENT persists the results and permits all features of Detect.  EPHEMERAL will be removed in the next release."
             )
```

Comment:
There shouldn't be any need to change the doc manually for deprecated properties/values. Just deprecating the value should take care of it. (I haven't seen this work yet for deprecated values, but I'd be surprised if it wasn't implemented that way.)

Reply:
Assuming that the deprecation call does the trick, then I'll revert this.  However it *looked* to me like the deprecation mechanism was more for the property (eg. detect.blackduck.scan.mode) than for the property "value" (EPHEMERAL STATELESS etc.).

Reply:
There are two mechanisms... one for properties, and one for enum values.

Reply:
It appears you have to use ... build().deprecateValue(...)   It seems to do the trick